core/domain/topic_services.py

```diff
@@ -401,8 +401,6 @@ def apply_change_list(topic_id, change_list):
                 elif (change.property_name ==
                       topic_domain.TOPIC_PROPERTY_LANGUAGE_CODE):
                     topic.update_language_code(change.new_value)
-                else:
```

Comment:
Can you explain why these  were removed as well?

Reply:
These and the lines below are removed as they are checked in TopicChange class. For example an invalid property for CMD_UPDATE_TOPIC_PROPERTY is checked here:
https://github.com/oppia/oppia/blob/624f6d9f2101147045d9cdc19f8e21699eccc57b/core/domain/topic_domain.py#L127